plugin/aws/ec2Instance.go

```diff
@@ -255,10 +260,7 @@ func (inst *ec2Instance) checkLatestConsoleOutput(ctx context.Context) (*ec2Inst
 }
 
 func (inst *ec2Instance) Delete(ctx context.Context) (bool, error) {
-	_, err := inst.client.TerminateInstancesWithContext(ctx, &ec2Client.TerminateInstancesInput{
-		InstanceIds: awsSDK.StringSlice([]string{inst.id}),
-	})
-	return false, err
+	return false, inst.Signal(ctx, "terminate")
```

Comment:
It's weird that there's two ways to do this. But not necessarily a bad idea.

Reply:
Yeah, opted for keeping things symmetrical w/ AWS docs